pysages/methods/metad.py

```diff
@@ -20,7 +20,9 @@
 
 class MetadynamicsState(NamedTuple):
     """
-    Attributes
+    Helper Metadynamics state
+
+    Parameters
```

Comment:
It looks like changing `Parameters` to `Arguments` might fix the duplicate warning issue. All the other changes done below are probably not needed?

Reply:
Yes, the `Parameters` vs `Arguments` fixes the duplicate issue.

The other changes are needed to document `Metadynamics` correctly.
The documentation goes before the `__init__` not after it.